src/components/appSwitch/AppSwitch.js

```diff
@@ -241,24 +238,31 @@ const AppSwitch = (props: LoadingProps) => {
 
   useEffect(() => {
     if (isMobileNative && DeepLinking.pathname) {
-      return props.navigation.navigate(DeepLinking.pathname.slice(1))
+      deepLinkingNavigation()
     }
   }, [])
 
   useEffect(() => {
     if (isMobileNative && DeepLinking.pathname) {
       DeepLinking.subscribe(() => {
-        return props.navigation.navigate(DeepLinking.pathname.slice(1))
+        deepLinkingNavigation()
       })
     }
+    return () => {
```

Comment:
could use one-liner here as cleanup hook doesn't have return value limitations as the effect